python-polars/README.md

```diff
@@ -0,0 +1,3 @@
+# Pandas-Groupby
+
+Supporting code for the article [Python Polars: A Lightening-Fast DataFrame Library](https://realpython.com/polars-python-lightening-fast-dataframe-library/).
```

Comment:
I suggest that we add a small paragraph about necessary dependencies as well. I think the readers need to do something like `python -m pip install "polars[numpy,pandas]" requests matplotlib` in order to run the examples.